friture/Levels.qml

```diff
@@ -75,6 +75,28 @@ Rectangle {
             Layout.alignment: Qt.AlignVCenter | Qt.AlignRight
             color: systemPalette.windowText
         }
+		
+		Text {
```

Comment:
If I read this correctly, the SPL value is shown even if there is no calibration file set. Could this be adjusted to only show the SPL values and the "dB SPL" label below when a calibration file is provided ?